deebot_client/hardware/deebot/ilt3k8.py

```diff
@@ -0,0 +1,240 @@
+    DataType.JSON,
+    Capabilities(
+        device_type=DeviceType.VACUUM,
+        availability=CapabilityEvent(
+            AvailabilityEvent, [GetBattery(is_available_check=True)]
+        ),
+        battery=CapabilityEvent(BatteryEvent, [GetBattery()]),
+        charge=CapabilityExecute(Charge),
+        clean=CapabilityClean(
+            action=CapabilityCleanAction(command=CleanV2, area=CleanArea),
+            continuous=CapabilitySetEnable(
+                ContinuousCleaningEvent,
+                [GetContinuousCleaning()],
+                SetContinuousCleaning,
+            ),
+            count=CapabilitySet(CleanCountEvent, [GetCleanCount()], SetCleanCount),
+            log=CapabilityEvent(CleanLogEvent, [GetCleanLogs()]),
+            work_mode=CapabilitySetTypes(
+                event=WorkModeEvent,
+                get=[GetWorkMode()],
+                set=SetWorkMode,
+                types=(
+                    WorkMode.MOP_AFTER_VACUUM,
+                    WorkMode.VACUUM,
+                    WorkMode.VACUUM_AND_MOP,
+                ),
+            ),
+        ),
+        custom=CapabilityCustomCommand(
+            event=CustomCommandEvent, get=[], set=CustomCommand
+        ),
+        error=CapabilityEvent(ErrorEvent, [GetError()]),
+        fan_speed=CapabilitySetTypes(
+            event=FanSpeedEvent,
+            get=[GetFanSpeed()],
+            set=SetFanSpeed,
+            types=(
+                FanSpeedLevel.QUIET,
+                FanSpeedLevel.NORMAL,
+                FanSpeedLevel.MAX,
+                FanSpeedLevel.MAX_PLUS,
+            ),
+        ),
+        life_span=CapabilityLifeSpan(
+            types=(
+                LifeSpan.BRUSH,
+                LifeSpan.CLEANING_SOLUTION,
+                LifeSpan.DUST_BAG,
+                LifeSpan.HAND_FILTER,
+                LifeSpan.FILTER,  # heap
+                LifeSpan.ROUND_MOP,
+                LifeSpan.SEWAGE_BOX,
+                LifeSpan.SIDE_BRUSH,
+                LifeSpan.STRAINER,
+                LifeSpan.UNIT_CARE,
+                LifeSpan.WATER_SINK,
+            ),
+            event=LifeSpanEvent,
+            get=[
+                GetLifeSpan(
+                    [
+                        LifeSpan.BRUSH,
+                        LifeSpan.CLEANING_SOLUTION,
+                        LifeSpan.DUST_BAG,
+                        LifeSpan.HAND_FILTER,
+                        LifeSpan.FILTER,  # heap
+                        LifeSpan.ROUND_MOP,
+                        LifeSpan.SEWAGE_BOX,
+                        LifeSpan.SIDE_BRUSH,
+                        LifeSpan.STRAINER,
+                        LifeSpan.UNIT_CARE,
+                        LifeSpan.WATER_SINK,
+                    ]
+                )
+            ],
+            reset=ResetLifeSpan,
+        ),
+        # TODO: add map once we get a working implementation
+        # see https://github.com/DeebotUniverse/client.py/pull/1102 for error logs
```

Comment:
```suggestion
        # TODO: add map once we get a working implementation
```

Don't refer a PR, which is implementing something else for logs. Please open a issue and add the error log there

Reply:
Done (#1109). Do you also want me to open an issue for true detect?